sim/target-agnostic.mk

```diff
@@ -43,6 +39,15 @@ midas_cc = $(shell find $(simif_dir) -name "*.cc")
 common_cxx_flags := $(TARGET_CXX_FLAGS) -Wno-unused-variable
 common_ld_flags := $(TARGET_LD_FLAGS) -lrt
 
+# Used as the prefix for all GG-generated output files, baked into the FPGA compilation flow.
+base_file_name := cl_firesim_generated
+
+# Simulation memory map emitted by the MIDAS compiler
+# TODO lowercase + better names?
```

Comment:
do you want to do this TODO now or are you intentionally punting for a future PR?

Reply:
I'd like to talk about it now, and remove it in this PR. Ill add a box for it.

Reply:
Did you do this?  I see the box you added in the PR description and it's still unchecked.